reference/enterprise-api-ref/inventory.markdown

```diff
@@ -0,0 +1,255 @@
+---
+layout: default
+title: Inventory API
+published: true
+tags: [reference, enterprise, API, reporting, URI]
+---
+Inventory API allows to access inventory reports and variable dictionary.
+
+## Inventory Reports
+   
+
+**URI:** https://hub.cfengine.com/api/inventory
+
+**Method:** POST
+
+
+**Parameters:**
+
+* **select** *(array)*
+    Fields for selecting. Required parameter.
+    List of fields name you can obtain through [List of variables][Inventory API#List of variables] call described below.
+* **filter** *(json object)*
+    Filter data. Format is ```{
+                                    "Attribute name":{  
+                                       "operator":"%value1%",
+                                       "operator":"%value2%"
+                                    }
+                                 }```. Optional parameter.
+* **sort** *(string)*
+    Field name for sorting with "-" for DESC order. Optional parameter.
+* **start** *(integer)*
+    Number of results to start from. Optional parameter.
+* **limit**  *(integer)*
+    Limit the number of results in the query. Default value is 1000, max value is 10000.
+* **hostContextExclude** *(array)*
+    Includes only results that concern hosts which have all specified CFEngine contexts (class) set. Optional parameter.
+* **hostContextInclude** *(array)*
+    Excludes results that concern hosts which have specified CFEngine context (class) set. Hosts that have at least one of the specified contexts set will be excluded from the results. Optional parameter.
+
+
+**Operators:**
+For filtering you can use operators bellow:
+
+| Abbreviation | Operator  |
+|--------------|-----------|
+| lt           | <         |
+| gt           | >         |
+| eq           | =         |
+| neq          | !=        |
+| lteq         | <=        |
+| gteq         | >=        |
+| l            | LIKE      |
+| nl           | NOT LIKE  |
+| il           | ILIKE     |
+| nil          | NOT ILIKE |
```

Comment:
Are these standard abbreviations?
I definitely would not have figured out what `l`, `nl`, `il`, and `nil` were related to database LIKE operators. I am also no API guru.

Reply:
There are no standards to do abbreviations with LIKE. I will change it. read the comment https://github.com/cfengine/documentation/pull/1613#discussion_r140219685